### PR TITLE
Create rishav.json

### PR DESCRIPTION
**Pull request by R15hav**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/rishav.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "portfolio domain",
+    "repo": "https://github.com/r15hav/r15hav.github.io",
+    "owner": {
+        "username": "r15hav",
+        "email": "r15hav@gmail.com",
+    },
+    "record": {
+        "CNAME": "r15hav.github.io"
```
Comment (CuteDog5695) [inline domains/rishav.json:9]: site seems to give 404 error, tho if you have site hosted then send link here

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
since site might be hosted and record is correct, ill merge it

**Review by iostpa — CHANGES_REQUESTED** [review]
cant merge it since repo link doesnt exist (ok now i think it wasnt the case here, more about the other thing i just requested changes)

File: domains/rishav.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "portfolio domain",
+    "repo": "https://github.com/r15hav/r15hav.github.io",
+    "owner": {
+        "username": "r15hav",
+        "email": "r15hav@gmail.com",
```
Comment (CuteDog5695) [inline domains/rishav.json:6]: just noticed this, remove the comma here

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/rishav.json (hunk 2 above)
Comment (VaibhavSys) [reply domains/rishav.json:6]:
```suggestion
        "email": "r15hav@gmail.com"
```

**Pull request closed without merging**